Sources/HaCWebsiteLib/ViewModels/LandingPage/EventFeature.swift

```diff
@@ -56,6 +60,11 @@ struct EventFeature: LandingFeature {
     return paddedEventPeriod.contains(currentDate)
   }
 
+  // Whether the event is on the same day
```

Comment:
To make this into a [Markup Formatted](https://developer.apple.com/library/content/documentation/Xcode/Reference/xcode_markup_formatting_ref/SingleLineComment.html#//apple_ref/doc/uid/TP40016497-CH102-SW1) documentation note, add another slash (i.e. `///`). This is so that tools like Xcode know that it's a user-facing documentation comment rather than a comment for people looking at the source file.

Reply:
Of course I totally forgot about this 😄